ortho_seq_code/gui/job_runner.py

```diff
@@ -0,0 +1,40 @@
+from PyQt5.QtWidgets import QWidget
+
+from ortho_seq_code.orthogonal_polynomial import orthogonal_polynomial
+from ortho_seq_code.gui.worker import Worker
+
+
+class JobRunner(QWidget):
+    def __init__(self, parent, threadpool):
+        super(QWidget, self).__init__(parent)
+        self.parent = parent
+        self.threadpool = threadpool
+
+    def job_func(self, progress_callback):
+        orthogonal_polynomial(
+            self.filename,
+            self.pheno_file,
+            self.molecule,
+            self.poly_order,
+            self.precomputed,
+            out_dir=".",
```

Comment:
I'm wondering if we can do something other than a '.' here? Then it isn't easy to see the outputs saved...